smoke-tests/framework/src/main/java/com/microsoft/applicationinsights/smoketest/SmokeTestExtension.java

```diff
@@ -689,4 +689,18 @@ public static Predicate<Envelope> getMetricPredicate(
       return name.equals(md.getMetrics().get(0).getName()) && isSecondPredicateValid;
     };
   }
+
+  public static Predicate<Envelope> getStandardMetricPredicate(String metricId) {
```

Comment:
```suggestion
  public static Predicate<Envelope> buildStandardMetricPredicate(String metricId) {
```

Reply:
The predicate could be a constant

Reply:
> 

prefer to be consistent with `getMetricPredicate` in that same class.